### PR TITLE
Refactor task cache-config

### PR DESCRIPTION
**Pull request by felixzapata**
If you use the package `glob-all` is not necessary to make a push to the array with the files. This way, you can add an array of patterns to the glob function.

Regards.

### CODE REVIEW

**Comment by samccone** [comment]
this is great good stuff @felixzapata

**Comment by samccone** [comment]
Landed in 
77b4ab90c179c33715d495877e44f1f9332f3077

thanks again :sparkling_heart:

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "connect-history-api-fallback": "^1.1.0",
     "del": "^1.1.1",
     "glob": "^5.0.6",
```
Comment (robdodson) [inline package.json]: it doesn't look like `glob` is being used by gulp any longer. Can we remove this @samccone ?

Comment (felixzapata) [inline package.json]: yes,  `glob` can be remove

**Pull request closed without merging**